src/test/regress/expected/pg16.out

```diff
@@ -127,6 +128,92 @@ HINT:  Consider specifying a name for the statistics
 CREATE STATISTICS (ndistinct, dependencies, mcv) on a, b from test_stats;
 ERROR:  cannot create statistics without a name on a Citus table
 HINT:  Consider specifying a name for the statistics
+-- STORAGE option in CREATE is already propagated by Citus
+-- Relevant PG commit:
+-- https://github.com/postgres/postgres/commit/784cedd
+SET citus.log_remote_commands TO on;
+CREATE TABLE test_storage (a text, c text STORAGE plain);
+SELECT create_distributed_table('test_storage', 'a');
+NOTICE:  issuing BEGIN TRANSACTION ISOLATION LEVEL READ COMMITTED;SELECT assign_distributed_transaction_id(xx, xx, 'xxxxxxx');
```

Comment:
instead of logging the whole commands sent, can we maybe check the actual states from all nodes for all the shards:

```SQL
 select result from run_command_on_all_nodes($$select array_agg(DISTINCT (attname, attstorage)) from pg_attribute  where attrelid::regclass::text ilike 't1%' and attnum> 0;$$);
```

so that we can use alter_distributed_table and/or undistribute_table as well with not too much outputs